keel-sql/src/main/resources/db/changelog/20201011-resource-debug-log.yml

```diff
@@ -0,0 +1,60 @@
+databaseChangeLog:
+  - changeSet:
+      id: resource-debug-log
+      author: emjburns
+      changes:
+        - createTable:
+            tableName: resource_debug
```

Comment:
A table without a primary or unique key is a table from which rows cannot be deleted without blocking inserts.